example/src/components/Typo.tsx

```diff
@@ -172,12 +178,14 @@ type TypoProps = {
   variant: VariantName;
   color?: string;
   children: ReactNode;
-};
+} & TextProps;
 
 export const Typo = ({
   variant = 'body-big',
   color = TextColors.darkText,
   children,
+  style,
```

Comment:
Are you sure that adding custom styling to `Typo` component is a good practice? I would have assumed that we should keep it clean from custom behaviour since it was predefined in the designs and all of the additional styling should be done around it using `<View>` components.

Reply:
Sg.